Add off boarding guide (#124)
MinBZK/ai-validation: docs/Way-of-Working/Offboarding.md

> # Off Boarding

Reviewer: Should we add a section about one of the following items:

1. Kubernetes / Digilab
2. Pleio community
3. Signal groups

We're sad to see you go! But if you do, here's what not to forget.

## Webex

- Leave our Webex Team.

> ## Github

Reviewer: ```suggestion
## GitHub
```

> - Make sure you handed any ownership you may still over to the team.

Reviewer: ```suggestion
- Make sure you transfer any ownership you may have to the team.
```

- Make sure to leave our team  [AI Validation Team](https://github.com/orgs/MinBZK/teams/ai-validation-team).
- If you are leaving MinBZK altogether, make sure to leave [MinBZK Org](https://github.com/orgs/MinBZK).

> ## Collaboration Space

Reviewer: ```suggestion
## Collaboration space
```

- Leave our collaboration space [Team Collaboration Space](https://www.samenwerkruimten.nl/teamsites/ai-validatie-team).

> ## Shared email

Reviewer: ```suggestion
## Shared mailbox
```

> - Make sure to leave our  [shared email](../../About/contact.md) by sending an email to Secretariat of Digital Society.

Reviewer: ```suggestion
- Make sure to leave our  [shared mailbox](../../About/contact.md) by sending an email to Secretariat of Digital Society.
```

> - Make sure to leave our  [shared email](../../About/contact.md) by sending an email to Secretariat of Digital Society.

Reviewer: @anneschuth is the email of the Secretariat of Digital Society a publicly known mailbox? If yes, I would propose to make it easy and include the email address here. Or can we refer to an internal address book where people can find the email adres?

## Teams page

- Move yourself to our alumni section on the [team page](../About/team.md).

> ## Mattermost Chat

Reviewer: ```suggestion
## Mattermost chat
```
